networking/ingress-controller-dnsmgt.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="ingress-controller-dnsmgt"]
+= Configuring an Ingress Controller for disabling DNS Management
+include::_attributes/common-attributes.adoc[]
+:context: ingress-controller-dnsmgt
+
+toc::[]
+
+As a cluster administrator, when you create an Ingress Controller, the Operator manages the DNS records by default. This has some limitations when the required DNS zone is different from the cluster DNS zone or when DNS zone is hosted outside the cloud provider. In such scenarios, the Operator reports a degraded state after attempting to provision the required resources in the cloud provider because the specified DNS can not be created.
+
+Day 2 operations add support to the Operator to have the option to manage DNS record lifecycle on the cloud provider. As a cluster administrator, you can create or modify an Ingress Controller with `dnsManagementPolicy` to specify whether the Operator needs to manage the DNS record lifecycle or not.
+You need this in instances where the required DNS zone is different from the cluster DNS zone and may not be hosted in the specific cloud provider.
+
+[NOTE]
+====
+This feature is supported on default or custom Ingress Controllers. As a clsuter administrator, you have the discretion to create or modify the default Ingress Controller.
```

Comment:
Could you please elaborate on the difference between non-default and custom? my thought is that when it is non-default, it means we are customizing it hence the word custom , unless i am not understanding.

Reply:
Non-default and custom mean the same, just that `non-default` is more explicit in what it conveys since this feature is NOT supported for the default ingress controller. 

If it makes it clear we can mention this the other way around, 
```
This feature is not supported on default Ingress Controllers 
...
```